bolt/lib/Passes/FixRISCVCallsPass.cpp

```diff
@@ -39,8 +41,19 @@ void FixRISCVCallsPass::runOnFunction(BinaryFunction &BF) {
         auto *Target = MIB->getTargetSymbol(*II);
         assert(Target && "Cannot find call target");
 
+        auto OldCall = *NextII;
```

Comment:
We've got a policy to not using auto if it is not necessary, please minimise their usage

Reply:
Let the Meta team members to clarify, they even has patches for auto expansion and asked me couple years ago. Maybe the policy changed. At least the LLVM coding standard says:
>> Don’t “almost always” use auto, but do use auto with initializers like cast<Foo>(...) or other places where the type is already obvious from the context. 
E.g. here it is obviously MCInst or MCSymbol above & etc. I don't really insist, but as for me I don't really like autos too except for complex types with smth like returning std::vector with std::pair elements..

Reply:
Yes, we should stick to LLVM policy of only using auto in small number of cases (the type is obvious (casts), the type doesn't matter (lambdas), template type). Making type explicit helps with reviewing where we have limited context window by default.